app/config_objects/district_config.rb

```diff
@@ -0,0 +1,69 @@
+class DistrictConfig
+
+  cattr_accessor :remote_filenames
+
+  ### CONFIGURING FILE NAMES FOR REMOTE DATA IMPORT ###
+
+  # To configure a Student Insights instance to read data from remote SFTP,
+  # we need to know what filenames to look for.
+
+  # Set filenames for the expected remote files in a .yml file under /config.
+  # See config/district_somerville.yml as an example.
+
+  # The file name should reference an ENV['DISTRICT_KEY'] value, like this:
+  #
+  # => "config/district_#{ENV['DISTRICT_KEY']}.yml"
```

Comment:
I'd suggest making this mapping manual and explicit like `{ somerville: district_somerville.yml }`.  There's no benefit to reducing duplication here, and it avoids using variables to form paths on the filesystem.

Reply:
Good suggestion, didn't realize using variables to form paths on the filesystem was a bad practice but now that I think about it that makes sense.